setup.py

```diff
@@ -9,7 +9,7 @@
 
 deps = [
     "zepben.auth==0.12.1",
-    "zepben.protobuf==0.32.0",
+    "zepben.protobuf==0.33.0b4+local2",
```

Comment:
todo: update this after https://github.com/zepben/ewb-grpc/pull/121 has been merged.